src/main/java/androidx/graphics/Path.kt

```diff
@@ -116,3 +116,18 @@ inline infix fun Path.xor(p: Path): Path {
         op(p, Path.Op.XOR)
     }
 }
+
+/**
+ * Create a path with closed shape from [points].
+ */
+inline fun createPathFromPoints(points: List<PointF>): Path {
```

Comment:
It doesn't sound efficient to first build a list of point to then create a path, when you can just create the path in the first place.

Reply:
for the case I have a list of point already, and want to build a path.